assignments/objects.js

```diff
@@ -11,34 +11,77 @@
 
 // Example format of an intern object: 1,examples@you.edu,Example,F
 const example = {
-  "id": 0,
+  "id": 1,
   "name": "Example",
   "email": "examples@you.edu",
   "gender": "F"
 }
-
+// console.log(example); 
 // Write your intern objects here:
+const mitzi = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+
+const kennan = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  speak() {
+    return ("Hello, my name is " + kennan.name + "!")
+  }
+}
+
+const keven = {
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
+
+const gannie = {
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+}
+
+const antonietta = {
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F", 
+  multiplyNums(a, b) {
```

Comment:
declaring the methods like this works, but the more traditional syntax is to do it like this

```
multiplyNums: function(a,b) {
return a*b;
}
```